tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -269,9 +269,111 @@
 //! The [`Subscribe::boxed`] method is provided to make boxing a subscriber
 //! more convenient, but [`Box::new`] may be used as well.
 //!
+//! When the _number_ of subscribers varies at runtime, note that a
```

Comment:
nit, feel free to ignore. not sure that we're contrasting the "number" from anything.

```suggestion
//! When the number of subscribers varies at runtime, note that a
```